internal/renderers/skia/lib.rs

```diff
@@ -120,6 +119,8 @@ pub struct SkiaRenderer {
     pre_present_callback: RefCell<Option<Box<dyn FnMut()>>>,
     partial_rendering_state: Option<PartialRenderingState>,
     visualize_dirty_region: bool,
+    /// Tracking dirty regions indexed by buffer age - 1. More than 3 back buffers aren't supported, but also unlikely to happen.
+    back_buffer_dirty_regions_by_age_minus_1: RefCell<[DirtyRegion; 3]>,
```

Comment:
There must exist a better name :-). Maybe `dirty_region_history`

Reply:
That is a better name - thanks :)